include/h2o/socket.h

```diff
@@ -130,6 +130,10 @@ void h2o_socket_dispose_export(h2o_socket_export_t *info);
  */
 void h2o_socket_close(h2o_socket_t *sock);
 /**
+ * Obtain the underlying fd of a sock struct
+ */
+int h2o_get_sock_fd(h2o_socket_t *sock);
```

Comment:
Please change the name to `h2o_socket_get_fd`.  I would like to have all the socket APIs have `h2o_socket_` as the prefix.